README.md

```diff
@@ -66,6 +66,15 @@ Provided you have correctly setup your [network configuration](https://doc.subst
 Make sure you deploy this backend with a compatible ecosystem (chaincode, hlf-k8s, etc).
 Always refer to the [compatibility table](https://github.com/SubstraFoundation/substra#compatibility-table).
 
+The recommended way to run a specific version (0.1.6) of substra-backend is to execute:
+
+```bash
+export SUBSTRA_BACKEND_VERSION=0.1.6
```

Comment:
export may not be needed like here : https://github.com/SubstraFoundation/hlf-k8s/pull/117/files#r560370670

Reply:
Thanks for the feedback, but unless I'm missing something it is indeed necessary (see my answer in the other PR).